test/integration/generated/azureSpecialProperties/src/operations/xMsClientRequestId.ts

```diff
@@ -30,13 +30,15 @@ export class XMsClientRequestId {
    * 9C4D50EE-2D56-4CD3-8152-34347DC9F2B0.
    * @param options The options parameters.
    */
-  get(options?: coreHttp.OperationOptions): Promise<coreHttp.RestResponse> {
+  getOperation(
```

Comment:
I see a lot of suffixing in places where it might not be semantically necessary, should we try to minimize that so that the operation (and operation group) names are less complicated?

Reply:
I minimized it by nor suffixing operations. However I had to keep suffixing OperationGroups